README.md

```diff
@@ -1,2 +1,30 @@
-# pallene-tracer
-Pallene Tracer is a mechanism and protocol for Lua C modules to have proper tracebacks
+<h1 align="center">The Pallene Tracer</h1>
+
+Pallene Tracer is a project providing protocols and underlying mechanism aiming to enable Lua C modules to have proper function call-traceback, without touching a single code in Lua internals!
```

Comment:
Instead of

> without touching a single code in Lua internals!

say that it works with vanilla Lua 5.4. Might also be worth emphasizing that Pallene Tracer isn't only for Pallene!